### DISCUSSION TITLE
An idea for monetization

### DISCUSSION THREAD

**Discussion by mirrorsedger** [RetroAchievements/RAWeb#1531] [Ideas]
I'm a fan of speedrunning and so I use the website [speedrun.com](speedrun.com) frequently.

That website offers a subscription model [(found here)](https://www.speedrun.com/supporter) which offers different kinds of perks for people that are willing to subscribe such as custom/animated usernames, name badges, boosts for different games etc.

I think something of a similar nature would be a good idea for RAWeb.
![Screenshot_1](https://user-images.githubusercontent.com/19656591/235311276-8fe59ff1-8769-4357-b77f-dcf1f7315b4b.png)

**Comment by wescopeland** [comment]
Why would it be a good idea?

**Reply by mirrorsedger** [reply]
My thinking was that if RA had subscriptions as a form of income, the income could be used to improve the service as a whole.

**Comment by Jamiras** [comment]
I'm sure there's someone better versed for responding to this, but we generally avoid collecting money as it could be used as leverage for the Big N to have us shut down.

We do have a patreon, and contributing to that gets you a badge on your profile page, but that's even set up to limit how many contribute so our income doesn't exceed our expenditures.

See also https://docs.retroachievements.org/FAQ/#is-retroachievements-profiting-off-of-emulation

**Reply by mirrorsedger** [reply]
I see. Perhaps the feautres listed could eventually be implemented. I thought that with better funding, RetroAchievements as a whole could be better maintained and more advanced features could be implemented that might currently be difficult to do. 

Thanks for the reply.

**Comment by televandalist** [comment]
For site features and whatnot, you could use something like Bountysource to place bounties on GitHub issues that get paid out to devs who resolve them. We had a handful of them in the past and a few of us, myself included, contributed to those bounties, but it wasn't something "official." I could see stuff like animated names, gif badges, and curated user lists (basically personal hubs for whatever) coming into play, but they'd probably be earned instead of being behind a paywall, similar to how our set requests work.

Something else that gets brought up every once in a while is achievement developers being paid to make sets. Personally, I think it ruins the motivation behind it and if there were an official system for it, would make things competitive to a nasty degree, especially if the numbers were public. A couple years ago, there was a dev who left a comment on their user wall saying that they'd only dev for Bitcoin from that point forward. I deleted the comment and bitched at them via DM over it. I don't think it's something that could be encouraged in good conscience. I also know of one set that exists because a user paid someone to do it, but it's just a rumor so there's not much to do about it. The most I could see happening in this regard would be some kinda tip system, like those "buy me a cup of coffee" links, but if the numbers were public like they are on Patreon, or if devs advertised/encouraged it, it'd be over. 

At one point, I actually did entertain the idea of setting up a personal Patreon for RA activities, via a suggestion. I ended up backing out of it because I felt my disclaimer would discourage donations, which defeated the purpose. I made it a point to say that there weren't any actual perks to it, that I wouldn't do things differently, give people special treatment, or prioritize some sets over others. 🤣

Anyway, there's already a big enough disconnect between casual users + the non-dev crowd and devs, especially when it comes to what we're even capable of doing with the toolkit. Throwing money into the mix would just make things worse and add layers of entitlement and obligations that no one signed up for.

**Comment by suspect15** [comment]
I'm completely opposed to monetization of RA in any form. What we have now is a passion project that is driven by good intentions. Monetization provides no valuable benefits but opens a Pandora's box of unwelcome problems. I don't see any realistic situation that would be anything anywhere close to a net positive for any users through monetization.

**Comment by meleu** [comment]
We had a RANews article addressing the monetization topic in detail.

Here's part of the article that seems to be related with this thread:

> keep in mind that increasing donations:
> 
> - helps us to pay the services needed to keep this project running.
> - shows how grateful the community is for Scott creating this project.
> - will **not** increase the number of new consoles being supported (this mostly happens in the [libretro](https://libretro.com/) front).
> - will **not** increase the number of creators working on new achievement sets.

[Check the full article here.](https://news.retroachievements.org/issues/2021-07/editorial.html)

If you really want to show your gratitude, just verbalize it wherever you can! Bring positive vibes. Use game’s official forum topic, or the achievement creator’s wall in his profile page. It can sound like a little thing, but I assure you that it causes a big positive impact. People usually are more vocal when they’re dissatisfied, and it usually puts a load of negativity in creators’ back. So, showing your gratitude with kind words is highly valuable.

**Comment by mirrorsedger** [comment]
Apologies everyone. My idea was simply to improve the service for everyone and reward the developers of RA for their work. However, I can now see why this is probably a bad idea for this community.

**Reply by meleu** [reply]
We believe your intentions are positive. :heart: 

> My idea was simply to improve the service for everyone and reward the developers of RA for their work.

Even if we ignore the "profiting from emulation" legal issue...

This project is what it is thanks to the effort of hundreds of contributors. There isn’t any reasonable way to fairly distribute any kind of financial support through the people who made/make RA.

You still have no-money ways to improve the service, though. You can check [How to contribute if you're not a developer](https://docs.retroachievements.org/How-to-contribute-if-you-are-not-a-developer/).

To reward developers, you can help them with tests, detailed reports to solve their tickets, or simply expressing your gratitude to them (this brings more benefit than money).

As you can see, it always boils down to "getting involved". Fortunately you can choose the degree of involvement you want to put. :slightly_smiling_face:

**Reply by mirrorsedger** [reply]
Thank you so much for your response. I love this community and I’ve already contributed on Patreon and made my own achievement set so I want us to continue improving on RA. I’ll try to contribute as much as I can :)
